.github/workflows/publishing.yml

```diff
@@ -380,23 +437,24 @@ jobs:
           ref: ${{ needs.assets.outputs.github_commit }}
           token: ${{ secrets.REPO_BOT_ACCESS_TOKEN }}
 
+      - name: Load wheel assets
+        uses: actions/download-artifact@v3
+        with:
+          name: wheel_assets
+          path: tmp/wheel_assets
```

Comment:
One other spot references `/tmp/wheel_assets` whereas the above line omits the leading slash, so I just wanted to confirm that this path doesn't need the leading slash, too? Same for line 451. (Maybe it doesn't matter ... I just wanted to double check.)

Reply:
Good catch. I can confirm that it worked either way (with or without the leading slash) but I've updated it to have the leading slash to be consistent. 

It could be the case that we just need to make the paths in 444 and 451 (consumer side) consistent, not necessarily the exact same path as the producer side.